UniversalMediaEngine/MediaEngine.h

```diff
@@ -5,76 +5,82 @@ using namespace Windows::Foundation;
 // Prototype
 class MediaEngineManager;
 
-namespace UniversalMediaEngine
-{
-	public enum class MediaEngineInitializationResult
-	{
-		Success,
-		Fail
-	};
-
-	public delegate void MediaStateChangedHandler(MediaState state);
-
-	public ref class MediaEngine sealed
-    {
-	private:
-		~MediaEngine();
-	public:
-
-		/// <summary>
-		/// Asynchronously initializes the media engine
-		/// </summary>
-		/// <returns> A task with a value that indicates whether Initialization was
-		/// successful</returns>
-		IAsyncOperation<MediaEngineInitializationResult>^ InitializeAsync();
-
-		/// <summary>
-		/// Plays the media stream at the given URL
-		/// </summary>
-		void Play(Platform::String^ url);
-
-		/// <summary>
-		/// Pauses playback
-		/// </summary>
-		void Pause();
-
-		/// <summary>
-		/// Stops playback
-		/// </summary>
-		void Stop();
-
-		property double Volume
-		{
-			/// <summary>
-			/// Gets the current volume
-			/// </summary>
-			/// <returns>The current volume</returns>
-			double get();
-
-			/// <summary>
-			/// Sets the current volume
-			/// </summary>
-			/// <param name="value">The new volume</param>
-			void set(double value);
+namespace Microsoft {
```

Comment:
I tried to do that but every time VS would just auto indent them for me so I gave up.